pkg/monitoring/metrics/virt-controller/vmistats_collector_test.go

```diff
@@ -365,20 +365,52 @@ var _ = Describe("VMI Stats Collector", func() {
 			}
 		},
 			Entry("no interfaces", [][]string{}),
-			Entry("one interface", [][]string{{"192.168.1.2", "InternalIP"}}),
+			Entry("one interface", [][]string{{"default", "192.168.1.2", "InternalIP"}}),
```

Comment:
default is not a bit confusing? what should be the default value here?

Reply:
default is literally the usual value of the primary interface name, that's why I used it here

Reply:
+1 🙏🏼